docs/develop/localhost.md

```diff
@@ -54,7 +54,56 @@ Then, as long as the container is running, you can invoke the model as an API as
 curl -X POST http://127.0.0.1:8080/v1/models/model:predict -d '{"inputs": [[0, 0, 0, 0]]}'
 ```
 
-### On (not) using Docker
+## Setting up local dev
+
+You have three options for how to run your Truss locally:
+
+* In a Docker container
+* In a Docker container with live reload
+* Without a Docker container in the Truss directory
+
+### Running with Docker
+
+This is the standard way to run a Truss. It creates a Docker image, runs the Truss, and returns a prediction. This method closely matches production environments, but results in a slow dev loop as it rebuilds the Docker container with every change to your code.
+
+To run in Docker:
+
+```python
+tr.docker_predict({"inputs": [[0, 0, 0, 0]]})
+```
+
+### Faster dev loop with live reload
+
+You can turn on a new feature, live reload, by setting the `live_reload` property to `True` in `config.yaml`. This feature makes it faster to run a Truss in Docker in some situations.
+
+It essentially works by keeping the Docker container running as you work on your Truss. Depending on the nature of the change, it may be able to keep the Docker container running; most changes do not affect the environment. This change is called a patch. Examples are changes to:
```

Comment:
Perhaps we can highlight the benefit a little better, something on the lines of `Depending on the nature of the change, it may be able to update the existing container, avoiding building a new docker image, making for a snappy development loop.`